Apps/Admin/Client/Program.cs

```diff
@@ -63,10 +66,23 @@ public static async Task Main(string[] args)
             // Enable Mud Blazor component services
             builder.Services.AddMudServices(config => config.SnackbarConfiguration.PositionClass = Defaults.Classes.Position.BottomRight);
 
+            builder.Services.AddScoped(_ => new HttpClient { BaseAddress = new Uri(builder.HostEnvironment.BaseAddress) });
+
+            WebAssemblyHost[] app = new WebAssemblyHost[1];
```

Comment:
Using an array here to avoid warnings/errors about variable initialization seems a little hacky, but I'm not sure how best to address that.